onboard/cv/main.cpp

```diff
@@ -70,15 +71,65 @@ void write_curr_frame_to_disk(Mat rgb, Mat depth, int counter) {
 int main() {
   /*initialize camera*/
   Camera cam;
+  cam.grab();
   int j = 0;
-  double frame_time = 0;
   int counter_fail = 0;
   #if PERCEPTION_DEBUG
-    namedWindow("image", 1);
+    namedWindow("Obstacle");
     namedWindow("depth", 2);
   #endif
   disk_record_init();
 
+  //Video Stuff
+  TagDetector d1;
+  pair<Tag, Tag> tp;
```

Comment:
NIT: tp = tag pair? This section could have more descriptive variable names, but just a nit